### PR TITLE
Cleanup Readme

### PR DESCRIPTION
**Pull request by erikh**
This incorporates #133, so please merge that first.

This adds simplified instructions based on the build system in #133 and additional cleanups such as word wrap and some whitespace changes.

### CODE REVIEW

**Comment by erikh** [comment]
@jainvipin can you review this change?

File: README.md [hunk 1]
```diff
@@ -12,117 +13,89 @@ Generic network plugin (experimental) is designed to handle networking use cases
...
+and few critical open issues are resolved.
+
+### Getting Started
+
+This will provide you with a minimal experience of uploading the intent and
+seeing the netplugin system act on it. It will create a network on your host
+that lives behind an OVS bridge and has its own unique interfaces.
+
+```
+$ git clone https://github.com/contiv/netplugin
```
Comment (jainvipin) [inline README.md]: @erikh - can we make 4 steps out of these commands?

File: README.md [hunk 2]
```diff
@@ -12,117 +13,89 @@ Generic network plugin (experimental) is designed to handle networking use cases
...
+$ cd netplugin; make build demo ssh
+< inside the vm >
+$ cd /opt/golang/src/github.com/contiv/netplugin
+$ sudo bin/netmaster &
+$ sudo bin/netplugin -host-label host1 &
+$ netdcli -cfg examples/one_host_multiple_nets.json
```
Comment (jainvipin) [inline README.md]: Feel free to mention a pointer to how the intent variations can look like

File: README.md (hunk 1 above)
Comment (erikh) [inline README.md]: ok

File: README.md (hunk 2 above)
Comment (erikh) [inline README.md]: Should I just point at the `examples/` directory?

Comment (jainvipin) [inline README.md]: This comment is not a significant one to incorporate. What you have is good, but leave an open exploration note to check out examples directory for more variations.

**Comment by erikh** [comment]
@jainvipin I have made the edits, PTAL.

File: README.md [hunk 3]
```diff
@@ -12,117 +13,108 @@ Generic network plugin (experimental) is designed to handle networking use cases
...
 
-`make demo`
+```
+$ git clone https://github.com/contiv/netplugin
+$ cd netplugin; make build demo ssh
+```
 
-Note:
-- Make sure VirtualBox is installed
+#### Step 2: Inside the VM, boot `netmaster` and `netplugin`
```
Comment (jainvipin) [inline README.md]: sorry I should have noticed this earlier. Do we need to bring them up explicitly? I thought you were going to incorporate this as part of make-demo. Or is that a separate PR? Even if it is, the doc could be updated now.

Comment (erikh) [inline README.md]: That's a separate bit of code I have yet to implement. For now, they need to start it themselves.

**Comment by erikh** [comment]
Going to merge this, I think it's satisfactory for now. Sorry if I jumped the gun.

**Comment by jainvipin** [comment]
LGTM

**Pull request merged**